cli/cliui/agent.go

```diff
@@ -357,10 +357,18 @@ type ConnDiags struct {
 	LocalNetInfo    *tailcfg.NetInfo
 	LocalInterfaces *healthsdk.InterfacesReport
 	AgentNetcheck   *healthsdk.AgentNetcheckReport
+	ClientIPIsAWS   bool
+	AgentIPIsAWS    bool
 	// TODO: More diagnostics
 }
 
 func ConnDiagnostics(w io.Writer, d ConnDiags) {
+	if d.PingP2P {
+		_, _ = fmt.Fprint(w, "✔ You are connected directly (p2p)\n")
```

Comment:
If they successfully connect p2p, I don't think we should (by default) throw up a bunch of p2p diagnostics with scary ❗s. It worked!, let's not worry them.

However, there might still be cases where p2p is intermittent, or a serious investigator is trying to compare and contrast different setups, so we can show everything if they run `coder ping -v`